stock_quant_manual_cost/models/stock_quant.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockQuant(models.Model):
+    _inherit = "stock.quant"
+
+    manual_cost = fields.Monetary(
+        compute="_compute_manual_cost",
+        store=True,
+        readonly=False,
+    )
+
+    @api.depends("inventory_diff_quantity", "product_id.standard_price")
+    def _compute_manual_cost(self):
+        for record in self:
+            record.manual_cost = record.product_id.standard_price
```

Comment:
Is it correct calculation? Should it be multiple with `inventory_diff_quantity`?